src/assets/audio/player.js

```diff
@@ -0,0 +1,17 @@
+let audio = new Audio('');
```

Comment:
I wonder if one small Vue component would do the trick here. Instead of plain js file